### ISSUE TITLE
Icon pack not applied to only one app

### ISSUE THREAD

**Issue by Fxsch** [MM2-0/Kvaesitso#543]
For some reason only Aurora Store uses the default icon instead of the icon pack

### Kvæsitso:
![Screenshot_20230914-204911_Kvaesitso_1](https://github.com/MM2-0/Kvaesitso/assets/77849217/fc6fbc76-63da-4519-9139-2a50dd57bc22)

### Lawnchair (correct icon):
![Screenshot_20230914-204925_Lawnchair_1](https://github.com/MM2-0/Kvaesitso/assets/77849217/e68aac0e-6a54-4b87-97fd-a489396e55e9)

**Comment by MM2-0** [comment]
Is it possible that you have changed the icon for that specific app in the past? Can you long press it, select 'customize', tap on the icon, and select the default icon again, just to be sure?

**Comment by Fxsch** [comment]
The default option is the wrong icon and I can't choose the lawnicons icon
![Screenshot_20230914-225540_Kvaesitso](https://github.com/MM2-0/Kvaesitso/assets/77849217/0b7def0f-a4c6-4074-8f7c-743aa7944586)

**Comment by exstntlmsnthrp** [comment]
@Fxsch what icon pack is that so others can try testing it?

**Comment by Fxsch** [comment]
It's [lawnicons](https://github.com/LawnchairLauncher/lawnicons). It's a build from their GitHub Actions tab.

**Comment by Fxsch** [comment]
I just noticed Aurora Store is not the only one. Happens with AL-chan too.

**Comment by MM2-0** [comment]
I think this issue is caused by Aurora Store and Lawnicons:

- Aurora Store recently [restructured their app](https://gitlab.com/AuroraOSS/AuroraStore/-/commit/544ee8eb54e3041a4ba3735b05d3ad899060c3fd) and renamed some activities in the process
- Now the activity names no longer match the names from the icon pack so the icons are not applied
- The lawnicon devs [reacted](https://github.com/LawnchairLauncher/lawnicons/commit/0c0d095e63a151c650bf9e0ec2f7841efad1656a) to this change and updated their icon pack accordingly, removing the old names in the process
- For some reason, F-Droid recommends an outdated version of Aurora Store (I guess the latest version is marked as a prerelease) and therefore doesn't auto update to the latest version


TL;DR: your Lawnicons and Aurora Store versions do not match, make sure you have both updated to the most recent (prerelease) version and it should resolve the issue.

And the reason it works in Lawnchair is that in the beginning, Lawnicons was using a different icon pack format (which matches icons solely on package names, not activity names, so it isn't affected by the activity renaming). Later they made the switch to the standard ADW launcher icon pack format. But apparently, that old format is still supported by Lawnchair (but only if you have "themed icons" enabled).

**Comment by Fxsch** [comment]
The issue with Aurora Store was fixed by updating the app. AL-Chan might have updated their Activity too because the one they use now is different from the one specified in Lawnicons. I'll just update that later.
